### DISCUSSION TITLE
explain for installation on esp8266 for powmr 10kw

### DISCUSSION THREAD

**Discussion by YAYASIMPSON** [syssi/esphome-smg-ii#55] [General]
hello is it possible to describe more what can install coorectly with homeassistant . because all of my test are bad... thanks a lot

**Comment by syssi** [comment]
Sure! Do you still need support?

**Comment by YAYASIMPSON** [comment]
hello 
i can't have any result : 
mqtt :

```
[C][mqtt.number:033]: MQTT Number 'smg-ii battery discharge recovery point in mains mode':
[D][modbus_controller:043]: Modbus command to device=1 register=0x196 countdown=0 no response received - removed from send queue
[W][modbus_controller:136]: Duplicate modbus command found: type=0x3 address=223 count=5
[D][modbus_controller:043]: Modbus command to device=1 register=0x6C countdown=0 no response received - removed from send queue
```

i haven't result on mqtt explorer or home assistant
i don't know if i have problem with flash or data in flash, for PowMR inverter, com is 2400 bauds and scan on modbus 5 so i change param

**Reply by syssi** [reply]
This log lines mean something like: "I try to talk to the device but the device doesn't respond". 

Could you provide some details about your wiring? Which YAML did you use? Which GPIOs do you use as TX/RX? Could you provide some photos? I would like to make sure the inverter is connected properly to the ESP.

**Reply by YAYASIMPSON** [reply]
this is a ESP8222 and I test with ESPWroom32 (i change the file sure ! )
this code is the last with ESP32 port TX;16/RX;17 and modbus 5 baudrate 2400.

![1713541941357](https://github.com/syssi/esphome-smg-ii/assets/96199989/c13bc0bb-3daf-4064-82b0-51cd29712bbd)
 ![pow_mr_yaml](https://github.com/syssi/esphome-smg-ii/assets/96199989/1697745a-6a13-43de-b7ba-63f590aa5cd4)
![1713541941373](https://github.com/syssi/esphome-smg-ii/assets/96199989/733b7793-2999-49ce-984f-b5c834feaf78)
